File: semester-5_vs/oop_lab-2/task-1_cpp/Place.cpp
#include "Place.h"

Place::Place(std::wstring name) :City(L"unknown") {
  this->placeName = name;
}

void Place::getInfo() {
  std::wcout << this->domainName << L"\n\r";
  std::wcout << this->metropolisName << L"\n\r";
  std::wcout << this->cityName << L"\n\r";
  std::wcout << this->placeName << L"\n\r";
  std::wcout << L"\n\r";
}

File: semester-5_replit/oop_task-2/Place.java
class Place extends City {
  public Place(String placeName) {
    super("unknown");
    this.placeName = placeName;
  }
  protected void getInfo() {
    System.out.println(domainName);
    System.out.println(metropolisName);
    System.out.println(cityName);
    System.out.println(placeName);
    System.out.println();
  }
  protected String placeName;
}